packages/stryker-mutator-specification/src/ArrowFunctionMutatorSpec.ts

```diff
@@ -0,0 +1,19 @@
+import { expect } from 'chai';
+import ExpectMutation from './ExpectMutation';
+
+export default function ArrowFunctionMutatorSpec(name: string, expectMutation: ExpectMutation) {
+  describe('ArrowFunctionMutator', () => {
+
+    it('should have name "ArrowFunction"', () => {
+      expect(name).eq('ArrowFunction');
+    });
+
+    it('should mutate an anonymous function with an inline return', () => {
+      expectMutation('const b = () => 4;', 'const b = () => undefined;');
```

Comment:
Does every language that we (plan) to support with this spec have arrow functions?

Reply:
Yes, the idea is that all "javascript like languages" should support mutation of arrow functions.

Reply:
I'll add it in the readme 👍